src/main/studio/kdb/K.java

```diff
@@ -101,6 +101,18 @@ public Adverb(K.KBase o) {
         public Object getObject() {
             return o;
         }
+
+        @Override
+        public String toString(boolean showType) {
+            try {
+                LimitedWriter writer = new LimitedWriter(256);
```

Comment:
Why the limit of 256? What happens e.g. if the iterated function is a long lambda?

Reply:
no good reason, took this limit from the one used by CellRenderer. But we could increase to something bigger - do you have a strong opinion?

Reply:
Does there have to be a limit at all? Or what if you give it INT_MAX?

Reply:
I'm afraid it may lead  to some sort of memory issue if the string generated is really large. I think we we can start with something like 5,000 or so. It can't be worse than today where it doesn't _output_ anything...

Reply:
Updated in last commit